examples/circular-callback-app.py

```diff
@@ -0,0 +1,64 @@
+import dash
+from dash.dependencies import Input, Output
+from dash import dcc, html
+import pandas as pd
+import plotly.express as px
+
+filepath = "https://raw.githubusercontent.com/plotly/datasets/master/volcano_db.csv"
+df = pd.read_csv(filepath, encoding="latin")
+
+external_stylesheets = ["https://codepen.io/chriddyp/pen/bWLwgP.css"]
+
+app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
+app.layout = html.Div([
+    html.Header("Geographic Distribution of Volcanoes by Height",
+                style={"font-size": "30px"}),
+    html.Div('Minimum Volcano Height',
+             style={"font-size": "20px"}),
+    'Meter ',
+    dcc.Input(
+        id="circular-callback-x-meter",
+        value=2000,
+        type="number"
+    ),
+    ' to Feet ',
+    dcc.Input(
+        id="circular-callback-x-feet",
+        value=6561.68,
+        type="number",
+    ),
+    dcc.Graph(
+        id="circular-callback-x-map"
+    )
+],
+    style={"marginLeft": 30, "marginTop": 30, "marginBottom": 30, "marginRight": 30})
+
+
+@ app.callback(
+    Output("circular-callback-x-meter", "value"),
+    Output("circular-callback-x-feet", "value"),
+    Output('circular-callback-x-map', 'figure'),
+    Input("circular-callback-x-meter", "value"),
+    Input("circular-callback-x-feet", "value"),
+)
+def sync_input(meter, feet):
+    ctx = dash.callback_context
+    input_id = ctx.triggered[0]["prop_id"].split(".")[0]
+    if input_id == "circular-callback-x-meter":
+        feet = None if meter is None else (float(meter) * 3.28084)
+    else:
+        meter = None if feet is None else (float(feet)/3.28084)
```

Comment:
We can use the updated `ctx` so can now be:
```
    if ctx.triggered_id == "circular-callback-x-meter":
        feet = None if meter is None else round((float(meter) * 3.28084),1)
    else:
        meter = None if feet is None else round((float(feet)/3.28084),1)

```

Given the topic of volcano height, I think it would be better to round to at most 1 decimal place.    The use case is to allow users to enter height in either meters or feet, rather than simply doing a conversion.  So the input labels could be "Meters"  and "Feet"

Reply:
okay I'll make a new PR since I had the time to implement your suggested changes now.